### PR TITLE
Mage_AdminNotification - DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
Clean #676

- fixed wrong translation call

### CODE REVIEW

File: app/code/core/Mage/AdminNotification/Model/Inbox.php [hunk 1]
```diff
@@ -131,7 +132,7 @@ public function parse(array $data)
     public function add($severity, $title, $description, $url = '', $isInternal = true)
     {
         if (!$this->getSeverities($severity)) {
-            Mage::throwException($this->__('Wrong message type'));
+            Mage::throwException(Mage::helper('adminnotification')->__('Wrong message type'));
```
Comment (kkrieger85) [inline app/code/core/Mage/AdminNotification/Model/Inbox.php]: Code change: but consistent to other translations in this class

Comment (sreichel) [reply app/code/core/Mage/AdminNotification/Model/Inbox.php]: Its a bug ... `__()` doesn't exit in model classes.

Comment (kkrieger85) [reply app/code/core/Mage/AdminNotification/Model/Inbox.php]: Therefore I approved this PR

**Comment by sreichel** [comment]
Bump. Easy to review.

**Pull request merged**